src/test/resources/config/application-artemis.yml

```diff
@@ -56,5 +56,8 @@ artemis:
     athena:
         url: http://localhost:5000
         secret: abcdef12345
+        modules:
```

Comment:
why do you define this both in application.yml and application-artemis.yml?

Reply:
I don't remember why I did that tbh. I removed it in `application.yml` and everything works as before.